woocommerce/woocommerce-gutenberg-products-block
# Add Atomic Category and Tag blocks

Introduces 2 new atomic blocks for the single product page; Tag and Category Lists.

Closes #2522 
Closes #2523

These blocks needed some API changes to expose the term data on the product endpoint.

Note, there is a known issue that blocks without data are still shown in the editor empty; this is being looked at by @LevinMedia 

### Screenshots

![Edit Page ‹ Local Testing — WordPress 2020-06-08 15-54-21](https://user-images.githubusercontent.com/90977/84045138-5a1e0600-a9a0-11ea-95b9-300396f1c1aa.png)
![Edit Page ‹ Local Testing — WordPress 2020-06-08 15-54-12](https://user-images.githubusercontent.com/90977/84045141-5ab69c80-a9a0-11ea-9434-216a436b769a.png)
![Edit Page ‹ Local Testing — WordPress 2020-06-08 15-54-00](https://user-images.githubusercontent.com/90977/84045143-5b4f3300-a9a0-11ea-8f59-12705fe7b31a.png)

### How to test the changes in this Pull Request:

1. Insert a single product block
2. Ensure the 2 new blocks are displayed in the layout
3. Save and check the same data is shown on the frontend
